src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -0,0 +1,1638 @@
+# coding=utf-8
+# Copyright 2021 Facebook AI Research The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2022 Facebook AI Research and The HuggingFace Inc. team. All rights reserved.
```

Reply:
Same above. The comment is resolved but the suggestion was not applied...